modules/zenith-public/sql/TraitsAndJAs.sql

```diff
@@ -111,6 +113,29 @@ UPDATE `abilities` SET `level` = 75, `content_tag`= NULL WHERE `name` = 'one_for
 UPDATE `abilities` SET `level` = 65, `content_tag`= NULL WHERE `name` = 'concentric_pulse';
 UPDATE `abilities` SET `level` = 71, `content_tag`= NULL WHERE `name` = 'theurgic_focus'; -- TODO: Not yet implemented. TODO: Potency changed from +50MAB to +25MAB
 
+-------------------------
+-- Content Tag Updates --
+-------------------------
+UPDATE `traits` SET `content_tag`= NULL WHERE `name` IN (
```

Comment:
<img width="721" height="39" alt="image" src="https://github.com/user-attachments/assets/76e6af6c-c2d3-4389-96f8-570f30654f93" />

Thought that looked funky. The table name is wrong

```suggestion
UPDATE `abilities` SET `content_tag`= NULL WHERE `name` IN (
```

Reply:
How embarrassing. Thanks for catching that. I pushed the correction.